clang/lib/Sema/SemaDecl.cpp

```diff
@@ -15501,8 +15511,10 @@ Decl *Sema::ActOnStartOfFunctionDef(Scope *FnBodyScope, Decl *D,
     FD->setInvalidDecl();
   }
   if (const auto *Attr = FD->getAttr<TargetVersionAttr>()) {
-    if (!Context.getTargetInfo().hasFeature("fmv") &&
-        !Attr->isDefaultVersion()) {
+    if (Context.getTargetInfo().getTriple().isRISCV()) {
```

Comment:
Should we check AArch64 in the original `if` instead?

Reply:
Updated.